airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -1036,13 +1039,15 @@ def __init__(
         aws_conn_id: str = "aws_default",
         waiter_countdown: int = 5 * 60,
         waiter_check_interval_seconds: int = 30,
+        force_stop: bool = False,
```

Comment:
WDYT about updating the `docs/apache-airflow-providers-amazon/operators/emr/emr_serverless.rst` to contain this additional param

Reply:
I added it to the system test (https://github.com/apache/airflow/pull/30757/files/8370c8172d8c28ffeec3ca460621b9d0447bfba6#diff-69be45953c5be696ca3159bb385a89c90930fc06e68357e8bdb33a1b31694f88R120) inside the `howto_operator` thing, so it'll be embeded in the doc automatically (as sample usage).
It doesn't explain what it's doing, but it being present there highlights its presence, and the code doc contains the explanation.

Reply:
looks good